src/sentry/integrations/github/client.py

```diff
@@ -40,7 +40,9 @@ def get_repo(self, repo):
 
     def get_repositories(self):
         repositories = self.get("/installation/repositories", params={"per_page": 100})
-        return repositories["repositories"]
+        repos = repositories["repositories"]
+        non_arch_repos = [repo for repo in repos if not repo["archived"]]
```

Comment:
@KKaempen Nit: You could always just return `[repo for repo in repos if not repo["archived"]]` directly without assigning to a variable

Reply:
Good call.